etl/sean.py

```diff
@@ -0,0 +1,68 @@
+'''
+Created on Dec 27, 2015
+@author: SeanEaster
+'''
+from math import floor
+
+
+import numpy as np
+import pandas as pd
+
+
+def standardize(array):
+    columnMeans = array.mean(axis=0)
+    return array - columnMeans
+
+# First, load the btc data and other assets
+btcDf = pd.DataFrame.from_csv('btc.csv')
+otherDf = pd.DataFrame.from_csv('data.csv')
+
+
+# running through different windows produces a lot of NaN values in rollingDeviations...this effs up np.divide later
+for _ in xrange(50):
+	STARTINDEX = np.random.randint(100)
+	LENGTH = np.random.randint(500)
+
+	btcValues = btcDf.ix[btcDf.index[STARTINDEX:(STARTINDEX + LENGTH)]].values.reshape((-1,))
+	meanBtcValues -= btcValues.mean()
+	btcStandardDev = meanBtcValues.std()
+
+	# Because numpy.correlate only supports single-dim arrays, we have to iterate
+	# Or I need to spend more time reading the docs. One of those two.
+
+	otherValues = otherDf.values
+	otherValues = standardize(otherValues)
+
+	rollingDeviations = pd.rolling_std(otherDf, LENGTH)#.values[(LENGTH - 1):, :]
```

Comment:
@seaneaster Here's where I'm running into trouble. It seems like in your version, you're limiting the bitcoin data to the start date and the end date and taking the standard deviation of this chunk. Then it looks like you're taking the full, unnormalized stock price list, taking a rolling standard deviation with the window being the length of the window we're looking at, and dividing the normalized list of stock price-btc correlations by a combination of this unnoramlized standard deviation-ed set and the btc standard deviations. 

I think my question, ATM, is why the rolling list of standard deviations? Why not just take the standard deviation of the stock prices over the window? My problem is that in my implementation, which you can see at etl/matcher.py, I'm limiting my data set to the time window, and then trying to take the rolling standard deviation over the length of the data set, which obviously doesn't work. I can't quite figure out what your'e doing though. Help?